.circleci/config.yml

```diff
@@ -124,6 +124,17 @@ jobs:
       - store_artifacts:
           path: fastlane/test_output/report.html
           destination: test_report.html
+  buildTvAndWatch:
```

Comment:
Heh he 🔨  📺. 👀

Reply:
let's add it to the list of jobs at the bottom of the file so it runs alongside tests